text_generation/causal_lm/cpp/continuous_batching/Dockerfile

```diff
@@ -5,7 +5,7 @@ WORKDIR /workspace
 RUN apt-get update -y && apt-get install -y python3-pip python3-venv git
 
 # Install OpenVINO
-RUN git clone https://github.com/openvinotoolkit/openvino.git && \
+RUN git clone --branch master https://github.com/openvinotoolkit/openvino.git && \
```

Comment:
can we use nightly package?
See example https://github.com/openvinotoolkit/openvino_tokenizers/pull/168

Reply:
We can add a flag to build container from git or from package. I can add this functionality in another PR.